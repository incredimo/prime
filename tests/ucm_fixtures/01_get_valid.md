```ucm get#g1 path="./README.md" range="1..10"
```