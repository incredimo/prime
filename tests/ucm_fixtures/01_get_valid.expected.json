{
  "id": "g1",
  "verb": "get",
  "attrs": {
    "path": "./README.md",
    "range": "1..10"
  }
}